workshops/weather_app/README.md

```diff
@@ -0,0 +1,384 @@
+---
+name: 'Weather App'
+description: 'Making a Weather App using HTML, CSS and JS(no framework involved)'
+author: '@gautamjajoo'
+---
+
+# Weather App
+
+Are you a beginner in JavaScript and don't know where to get started? If yes, you're in the right place! Being a beginner in coding and having to learn the fundamentals can be very painful—but the best way to learn is to make fun, simple projects! In this workshop, you'll build an app that uses web APIs to get the weather for any city.
+
+Here's what the final version looks like:
+
+![main](https://cloud-fw0yxvsf2.vercel.app/0screenshot_from_2020-10-23_19-26-13.png)
+
+The [Source Code](https://github.com/gautamjajoo/BasicWeather) & [Demo](https://WeatherApp.gautamjajoo.repl.co) are hyperlinked here.
+
+By the end of this workshop, you'll have learned how to use web APIs, which you could expand to use even more creatively in future projects. You'll also learn some fundamentals of JavaScript, as well as some nice CSS tricks for making beautiful card designs.
+
+## 1. The Prerequisites
+Some basic understanding of HTML,CSS and Javascript/JSON.
+
+If you don't know any of the above, don't worry! This workshop will guide you through it.
+
+## 2. Setting up the project on Repl.it
+
+We're going to be using [Repl.it](https://repl.it) for this workshop. Repl.it is a powerful collaborative online code editor.
+
+Start by creating a [new HTML/CSS repl.it project](https://repl.it/languages/html).
+
+You should see three files: `index.html`, `style.css`, and `script.js`.
+
+![repl.it screen](https://cloud-o4m9cgxr8.vercel.app/0screenshot_from_2020-10-09_12-14-29.png)
+
+## 3. Making a skeleton of the app.
+
+Let's start by adding some code to the `index.html` file. We start the file with some basic lines which are added in every HTML file.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Weather</title>
+</head>
+<body>
+</body>
+</html>
+```
+
+Here `!DOCTYPE` is used as an "information" to the browser about what document type to expect.
+
+Now we would link the `CSS` and `JS` files to our `HTML` file. Also, we would be using `normalize.css` which is a small CSS file used for styling 

```

Comment:
Kinda weird grammar here. Also if this workshop is aimed toward beginners, it’s very unclear to me what `normalize.css` is or what that means